pkgs/applications/networking/novnc/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Z+bks7kcwj+Z3uf/t0u25DnGOM60QhSH6uuoIi59jqU=";
   };
 
+  patches = [ ./fix-paths.patch ];
+
+  postPatch = ''
+    sed -i 's|@out@|'"''${out}/share/webapps/novnc"'|g' utils/novnc_proxy
```

Comment:
There are better forms for doing this. Given that you introduced `@vars@`, you can use `substituteAll`, a Nixpkgs function that does exactly this.

https://github.com/NixOS/nixpkgs/blob/417b476db7d319b4c981e2e1feb399b05bcd4ea7/pkgs/build-support/substitute/substitute-all.nix#L5

Reply:
Sorry, I need to sleep now...